### ISSUE TITLE
Erratic or No Rendering Using IPython 2.1.0 Interact decorator

### ISSUE THREAD

**Issue by pybokeh** [yhat/ggplot#342]
Hi, this is hard to explain my issue.  Basically, if I use ipython notebook's interact decorator, my ggplot doesn't render until I generate a plot using another library like seaborn.  When I execute the cell containing the code to generate the ggplot chart, the dropdown widget shows up, but not the actual chart.  But if I create a seaborn plot then execute the ggplot code, the ggplot chart then renders.  But there's more.  The first time I execute the seaborn plot, the plot is not in a seaborn style/theme but ggplot theme.  But if I run the code again, I get a warning about missing Arial font, then the seaborn plot renders in seaborn theme.  I think the missing font warning is a separate issue or concern not related or causing my ggplot to render.

So basically, I can get the ggplot to render with the interact decorator only after generating a seaborn plot first.

Here is my example ipython [notebook](http://nbviewer.ipython.org/gist/pybokeh/19eab1c278af1c971dee)

The sample wind turbine data can be obtained [here](https://github.com/wrobstory/pydatasv2014/blob/master/USGS_WindTurbine_201307_cleaned.csv)

My setup:
OS: Elementary OS Luna
Python version: Python 3.4 in virtual environment
IPython version 2.1.0
ggplot version 0.5.8
seaborn version 0.3.1
Browser: Chromium version 34.0.1847.116
